includes/core/Enums/class-wc-pay-endpoints.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace WCPay\Core\Enums;
+
+final class Wc_Pay_Endpoints
```

Comment:
Let's just use `Endpoints` as th class name, and importantly, try to either stick to `WCPay` (for the core namespace) or `WC_Payments` (the existing one for standard includes), but not add `Wc_Pay` or `WC_Pay`.

Reply:
Noted.